src/components/Calculator.js

```diff
@@ -1,42 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
+
+function Button({ onClick, label }) {
+  return (
+    <button type="button" onClick={onClick}>
+      {label}
+    </button>
+  );
+}
+
+Button.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  label: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] @Nezerwa its not good practise having two components in one file, kindly create a button component and move it there, this will make your code neat.